README.md

```diff
@@ -108,6 +108,43 @@ Install this project with:
   </ul>
 </details>
 
+### Usage
+
+To run the project, execute the following command:
+
```

Comment:
- [ OPTIONAL ] Since this project does not need to be run by any command, I think you should show a new developer how to use the project here (What to do to run the project ) :+1:

Reply:
A pleasure to work with you this week, dear friend @EddxSotz ... All the best, man  🙂🙂🤝🤝🙏🙏